test/real_cases_periodic_solver_test.rb

```diff
@@ -23,12 +23,15 @@ def test_periodic_and_ortools
       vrps = TestHelper.load_vrps(self)
 
       vrps.each{ |vrp|
+        Models.delete_all # needed to prevent duplicate ids
+        Helper.load_models([{vrp: vrp}])
```

Comment:
Brainstorming: I don't know if it is a good idea but what if we had a `delete_all` at the beginning of `load_models`? It looks like we always call them together. What do you think?

If we try to load a model over another one, we risk generating duplicate id error. I don't know if we would ever need to load two completely separate VRPs (i.e., no overlapping ids) one day but even so we can do so by giving both VRPs in an array like you did for job_manager.